tasks/main.yml

```diff
@@ -72,6 +72,7 @@
     state: absent
   with_items: "{{ project_directory_files }}"
   when: code_source == 'local'
+  become: true
```

Comment:
Paramaterise this with 

```
become: "{{ docker_become }}"
```
like how it's done elsewhere.

You also need to do this for the steps below this which clears directories.